.travis.yml

```diff
@@ -29,8 +29,12 @@ script:
     # Run unit tests and doc tests in debug mode
     - cargo test -p lumol-core
     - cargo test -p lumol-input
+    # Check if tutorials compile
+    - cd
     # Check that benchmarks still compile
     - cargo bench --no-run
+    # Check if tutorials compile
+    - ./scripts/ci/compile-tutorials.py
```

Comment:
I would rather explicitly do `cd tutorial/lumol_potential_tutorial && cargo build` here. We can craft a script for this later!

You might need to do a `cd $TRAVIS_BUILD_DIR` after that.

Reply:
Sure, that was my first attempt :)

Reply:
There is another way to do this, in one line: `cargo build --manifest-path=tutorial/lumol_potential_tutorial/Cargo.toml`.

But if you prefer the python script, you'll need to `chmod +x` it. There is a permission denied error on Travis.